include/gz/math/detail/Box.hh

```diff
@@ -310,6 +310,21 @@ bool Box<T>::MassMatrix(MassMatrix3<T> &_massMat) const
   return _massMat.SetFromBox(this->material, this->size);
 }
 
+/////////////////////////////////////////////////
+template<typename T>
+std::optional< MassMatrix3<T> > Box<T>::MassMatrix() const
+{
+  gz::math::MassMatrix3<T> _massMat;
+
+  if(!_massMat.SetFromBox(this->material, this->size))
+  {
+    return std::nullopt;
+  }
+  else
+  {
+    return std::make_optional(_massMat);
```

Comment:
include <optional>

Reply:
Hey @ahcorde, not sure what you mean, can you please clarify?

Reply:
ups, I edited the comment, But we need to include `<optinal>` at the top

Reply:
@ahcorde I have made the required changes